src/Lykke.Service.PaySettlement.Services/TransferToMerchantService.cs

```diff
@@ -0,0 +1,125 @@
+﻿using Autofac;
+using Common;
+using Common.Log;
+using Lykke.Common.Log;
+using Lykke.MatchingEngine.Connector.Models.Api;
+using Lykke.Service.Assets.Client.Models;
+using Lykke.Service.ExchangeOperations.Client;
+using Lykke.Service.ExchangeOperations.Client.AutorestClient.Models;
+using Lykke.Service.PaySettlement.Core.Domain;
+using Lykke.Service.PaySettlement.Core.Services;
+using Lykke.Service.PaySettlement.Core.Settings;
+using System;
+using System.Threading.Tasks;
+using System.Timers;
+
+namespace Lykke.Service.PaySettlement.Services
+{
+    public class TransferToMerchantService : IStartable, IStopable
+    {
+        private readonly ITransferToMerchantQueue _transferToMerchantQueue;
+        private readonly ILog _log;        
+        private readonly TransferToMerchantServiceSettings _settings;
+        private readonly IAssetService _assetService;
+        private readonly ILykkeBalanceService _lykkeBalanceService;
+        private readonly IStatusService _statusService;
+        private readonly IExchangeOperationsServiceClient _exchangeOperationsServiceClient;
+        private readonly Timer _timer;
+
+        public TransferToMerchantService(ITransferToMerchantQueue transferToMerchantQueue,
+            TransferToMerchantServiceSettings settings, IAssetService assetService, 
+            ILykkeBalanceService lykkeBalanceService, ILogFactory logFactory,
+            IStatusService statusService, IExchangeOperationsServiceClient exchangeOperationsServiceClient)
+        {
+            _transferToMerchantQueue = transferToMerchantQueue;
+            _log = logFactory.CreateLog(this);
+            _settings = settings;
+            _assetService = assetService;
+            _lykkeBalanceService = lykkeBalanceService;
+            _statusService = statusService;
+            _exchangeOperationsServiceClient = exchangeOperationsServiceClient;
+
+            _timer = new Timer(settings.Interval.TotalMilliseconds);            
```

Comment:
Do we need `Common.TimerPeriod` here?
(Timer that runs Execute method in a loop with a fixed time interval between runs (Execute method execution time is not included))